src/objects.js

```diff
@@ -1,52 +1,84 @@
 const createPerson = (name, age) => {
-  // your code here
+    const user = {
+        name: (name),
+        age: (age)
+    }
+    return user
 };
 
 const getName = (object) => {
-  // your code here
+    return object.name
+        // your code here
 };
 
 const getProperty = (property, object) => {
-  // your code here
+    return object[property]
+        // your code here
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+    const a = !!object[property]
+    return a
 };
 
 const isOver65 = (person) => {
-  // your code here
+    if (person.age > 65) {
+        return true
+    } else {
+        return false
+    }
 };
 
 const getAges = (people) => {
-  // your code here
+    return people.map(obj => obj.age)
+        // your code here
 };
 
 const findByName = (name, people) => {
-  // your code here
+    return people.splice(obj => obj.name = name).filter(obj => obj.name === name).reduce((obj, item) => {
```

Comment:
The array method we are looking for here is [Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find). 

I'd look up [Array.prototype.splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) as this doesn't look like correct usage.

It is also considered bad practice to return assignment: `obj.name = name`.